news/2022-31-07-osulazer-audio-design.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "sound design of osu!(lazer)"
+date: 2022-07-31 00:00:00 +0000
+needs_cleanup: true
```

Comment:
this only makes sense for existing articles, not newsposts, as it implies the contents are going to be rewritten later on

Reply:
Removed, thanks